client/src/reducers/revisions/cont.js

```diff
@@ -1,31 +1,13 @@
-import { Map } from 'immutable';
-
-export default ( state = new Map(), action ) => {
+export default ( state = '', action ) => {
 	switch ( action.type ) {
-		case 'REVISIONS_FETCH':
-			return action.users.reduce( ( set, user ) => {
-				if ( set.has( user ) ) {
-					return state;
-				}
-
-				return set.set( user, null );
-			}, state );
 		case 'REVISIONS_ADD':
-			return state.merge( action.cont );
-		case 'QUERY_USER_CHANGE':
-			return state.filter( ( cont, user ) => {
-				return action.users.includes( user );
-			} );
+		case 'REVISIONS_CONTINUE_SET':
```

Comment:
I couldn't find 'REVISIONS_CONTINUE_SET' nor 'REVISIONS_CONTINUE_DELETE' in use. Do we need them?

Reply:
Removed. :)